dotCMS/src/integration-test/java/com/dotcms/contenttype/business/RelationshipAPITest.java

```diff
@@ -231,6 +244,132 @@ public void testconvertRelationshipToRelationshipField_SelfRelated_Success() thr
         }
     }
 
+    /**
+     * <b>Method to test:</b> {@link RelationshipAPI#dbRelatedContent(Relationship, Contentlet)} <p>
+     * <b>Given Scenario:</b> A content is related to another (different content types) <p>
+     * <b>ExpectedResult:</b> {@link RelationshipAPI#dbRelatedContent(Relationship, Contentlet)} should always return related content
+     * @throws DotDataException
+     * @throws DotSecurityException
+     */
+    @Test
+    public void testDbRelatedContentWhenNonSelfJoinRelationship()
```

Comment:
Codacy found an issue: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5775242)